frame/core-fellowship/src/benchmarking.rs

```diff
@@ -0,0 +1,200 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Salary pallet benchmarking.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+use crate::Pallet as Salary;
+
+use frame_benchmarking::v2::*;
+use frame_system::{Pallet as System, RawOrigin};
+use sp_core::Get;
+
+const SEED: u32 = 0;
+
+fn ensure_member_with_salary<T: Config<I>, I: 'static>(who: &T::AccountId) {
```

Comment:
Just an FYI in case this wasn't clear (will prob add an additional doc comment clarifying this), but with benchmarking v2, it is completely valid to define a helper function like this, or really any other random code that isn't a benchmark, _inside of_ the `benchmarks` module i.e. the module that has `#[benchmarks]` or `#[instance_benchmarks]` attached to it. It is in every respect a normal module, that just happens to also have some `#[benchmark]`-marked functions in it.